requirements.txt

```diff
@@ -1 +1,2 @@
-databricks-api==0.9.0
\ No newline at end of file
+databricks-sdk
```

Comment:
Do we want to restrict this job to a specific version?

Reply:
Yes, done